frontend/components/commons/filters/SelectFilter.vue

```diff
@@ -93,7 +93,7 @@ export default {
       }
       let filtered = Object.fromEntries(
         Object.entries(options).filter(([id, value]) =>
-          id.match(text.toLowerCase())
+          id.toLowerCase().match(text.toLowerCase())
```

Comment:
@leiyre `text` is a selected option? In that case, could we just match the original values?

```javascript 
id.match(text)
```

Reply:
@frascuchon,`text` is any of available options, if we want case sensitive we can use `id.match(text)`. Is it ok case sensitive?